packages/cfd/src/Utils/Validator/validator.ts

```diff
@@ -1,9 +1,54 @@
 import { template } from '_common/utility';
 import { getPreBuildDVRs } from '@deriv/shared';
 import Error from './errors';
+import { TCoreStores } from '@deriv/stores/types';
+
+type Rule = {
```

Comment:
what is the difference between `Rule` and  `TRuleString` since both are objects with `name: string` and optional `options`

should we need to start every type definition with T? (i.e. TRule, TValidationFunction, TRuleObject...)